examples/common/chip-app-server/server.cpp

```diff
@@ -0,0 +1,96 @@
+#include "server.h"
```

Comment:
Should this file be called `Server.cpp` and similar for the `.h`?  Most CHIP code seems to follow that style...